src/upgrades/TEMPLATE

```diff
@@ -6,7 +6,8 @@ var async = require('async');
 var winston = require('winston');
 
 module.exports = {
-	name: 'User_friendly_upgrade_script_name',
+	name: 'User friendly upgrade script name',
```

Comment:
This is underscored so you can double-click to select the whole text

Reply:
Drat!